src/screens/WalletDetailsScreen.tsx

```diff
@@ -42,10 +42,14 @@ interface Props {
 
 export class WalletDetailsScreen extends React.PureComponent<Props> {
   componentDidMount() {
+    this.checkSubscription();
+  }
+
+  checkSubscription = () => {
     const { wallet, email, checkSubscription } = this.props;
     if (!email || !wallet) return;
```

Comment:
I know it's not your changes, but maybe let's change it to:
```
if (email || wallet) {
  checkSubscription([wallet], email)
}
```
?

Reply:
you meant, will add
```
if (email && wallet) {
  checkSubscription([wallet], email)
}
```

Reply:
done